### ISSUE TITLE
v2.1.0  Features

### ISSUE THREAD

**Issue by zoldello** [LorenFrankLab/rec_to_nwb_yaml_creator#35]
Fixes

- Fixed issue where bad channels were not being update on upload of a file (Sharon)
- Make UCSF as default Institution (Sam/Ryan/Eric)
- Added grams to subject-weight, in parenthesis (Sam/Ryan/Eric)
-  Device object NOT REMOVED per Eric's request for concern about rec-to-nwb crashing (Sam/Ryan/Eric)
- Make associated video file camera id to be radio button (Sam/Ryan/Eric)
- Species list shortens to the most common ones in neuroscience (Sam/Ryan/Eric)
- Associated Video Files - camera ID is now radio button (Sam/Ryan/Eric)
- Change download icon (Sharon)
- Datalist appear in all evergreen browsers (we do not support unsupported browsers like IE6). Firefox issue mainly (Sam)
- Task epochs could use a description (done via info icon)  (Sam/Ryan/Eric)
- Raw Data to Volts (could be more specific than Raw data, ephys conversion factor to convert ephys data to volts) (Sam/Ryan/Eric)
- Behavioral Events has Type and index added. A tooltip was placed in the info icon to explain the name. (Sam/Ryan/Eric)
- Changed Associated Video File -> task_epochs to task_epoch and made the value a integer from an array (Sam/Ryan/Eric)
- Fixed issue where the same yaml file could not be uploaded multiple times. (Phil)
- Fixed issue where missing key in yaml file caused the app to crash if the key is manually added. (Phil)

**Comment by edeno** [comment]
> + Species list shortens to the most common ones in neuroscience
I would say to shorten this to the most common ones used in the lab

**Comment by zoldello** [comment]
> > * Species list shortens to the most common ones in neuroscience
> >   I would say to shorten this to the most common ones used in the lab

We use only one species. So, I added a few more. When released, I can modify it if you object.

**Comment by zoldello** [comment]
@samuelbray32 Please, the changes you, Ryan and Eric requested and listed in the ticket were done. I did not address all your comments. I will address them in 1 or 2 more releases.
